app/controllers/admin/channels_controller.rb

```diff
@@ -3,16 +3,36 @@ class ChannelsController < AdminController
     include Search
 
     def index
-      @channels = Channel
+      @channels = if sort_column&.to_sym&.in? Channel::ADVANCED_SORTABLE_COLUMNS
+        Channel.advanced_sort(sort_column.to_sym, sort_direction)
+      else
+        Channel.order("#{sort_column} #{sort_direction}")
+      end
 
       if params[:q].present?
         query = params[:q].split(' ').map { |q| "%#{remove_prefix_if_necessary(q)}%" }.join(' ')
         @channels = @channels.search(query)
       end
 
+
       @channels = @channels.verified if params[:verified].present?
 
-      @channels = @channels.order(created_at: :desc).paginate(page: params[:page])
+      case params[:type]
+      when 'website'
+          @channels = @channels.site_channels
+      when 'youtube'
+          @channels = @channels.youtube_channels
+      when 'twitch'
+          @channels = @channels.twitch_channels
+      end
+
+      @channels = @channels.paginate(page: params[:page])
```

Comment:
We might need a `@channels.group(:id)` to prevent duplicates.

Reply:
I'm not sure if we allow duplicate channels but it could hurt to ensure none pop up

https://github.com/brave-intl/publishers/pull/1472/commits/8b2d8a77652d756d19fb8c801e09a8a43c5fcf08